examples/basic-datasource/.webpack/webpack.dev.conf.js

```diff
@@ -0,0 +1,203 @@
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ReplaceInFileWebpackPlugin = require('replace-in-file-webpack-plugin');
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const { getPackageJson, getPluginId, hasReadme } = require('./utils');
+const { SOURCE_DIR, DIST_DIR, ENTRY_FILE } = require('./constants');
+
+module.exports = {
+  mode: 'development',
+
+  // Compile for browsers
+  target: 'web',
+
+  // The "root dir" for the source code
+  context: path.join(process.cwd(), SOURCE_DIR),
+
+  // Source map type
+  devtool: 'source-map',
+
+  entry: {
+    module: path.resolve(process.cwd(), ENTRY_FILE),
+  },
+
+  output: {
+    filename: '[name].js',
+    path: path.resolve(process.cwd(), DIST_DIR),
+    libraryTarget: 'amd',
+    publicPath: '/',
+  },
+
+  // Don't show hints (e.g. asset size is over 250kb)
+  performance: { hints: false },
+
+  // External packages
+  // (these are going to be provided by the Grafana runtime)
+  externals: [
```

Comment:
is this plan to suggest everyone has their own copy of the config?  

Is there any way to help from getting out of sync?

For all the problems of toolkit... it did solve having 40 different, but almost identical configurations

Reply:
This is a great question and we are leaning towards stop recommending toolkit. Would be suuper helpful if you can add your feedback and experience on this topic in the following discussion: https://github.com/grafana/grafana/discussions/40568

All learnings and feedback is welcome and will help us take a better decision on this moving forward 🙏🏻  It would be really good to understand the dangers of getting out of sync (as you see it) with the build configurations.